Automattic/wp-calypso
# Replaced Growth image (ecommerce trial page)

## Proposed Changes

* Replaced wrong (repeated) image file.

## Testing Instructions
* Open the Plans page of a trial site and check the new image for the Marketing feature

* Before:
![image](https://user-images.githubusercontent.com/3801502/218087553-9686df2d-c61d-4406-a0f0-7c250f958daf.png)

* After:
![image](https://user-images.githubusercontent.com/3801502/218087690-334ee325-25ae-40e2-920c-b336267a5afe.png)
